server/auth_function/readme.md

```diff
@@ -35,6 +35,7 @@ sudo dockerd
 cd server/auth_function
 pytest
 ```
+Note: We need to use master db user(postgres) to run the tests.
```

Comment:
currently the auth function needs access to fam_user and fam_role table. Later one once we read admin role, we need access to admin tables. So only master db user can have access to all those tables.